File: Dubi-C#/Clases/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Clases
{
    public class Usuario : Natural
    {
        protected int idUsuario;
        protected int tipoUsuario;
        protected String password;
        protected int estadoU;

        public Usuario() { }
        public Usuario(string idPersona, string email, string telefono, string dni, string nombre, string apPat, string apMat, char sexo, DateTime fechaNac, int idUsuario, int tipoUsuario, string password, int estadoU)
        : base(idPersona, email, telefono, dni, nombre, apPat, apMat, sexo, fechaNac)
        {
            this.idUsuario = idUsuario;
            this.tipoUsuario = tipoUsuario;
            this.password = password;
            this.estadoU = estadoU;
        }

        public int IdUsuario { get => idUsuario; set => idUsuario = value; }
        public int TipoUsuario { get => tipoUsuario; set => tipoUsuario = value; }
        public string Password { get => password; set => password = value; }
        public int EstadoU { get => estadoU; set => estadoU = value; }
    }
}

File: Dubi-Java/src/Modelo/Usuario.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Modelo;

import java.sql.Date;

/**
 *
 * @author alulab14
 */


public class Usuario  extends PersonaNatural{
    
    private int idUsuario;
    private int tipoUsuario;
    private String password;
    private int estadoU;
    
    
    public Usuario(){
    }
    
    public Usuario(String idPersona, String email, String telefono, String dni, String nombre, String apPat, String apMat, char sexo, Date fechaNac, int idUsuario, int tipoUsuario, String password, int estadoU){
        super(idPersona, email, telefono, dni, nombre, apPat, apMat, sexo, fechaNac);
        
            this.idUsuario = idUsuario;
            this.tipoUsuario = tipoUsuario;
            this.password = password;
            this.estadoU = estadoU;
    }

    /**
     * @return the idUsuario
     */
    public int getIdUsuario() {
        return idUsuario;
    }

    /**
     * @param idUsuario the idUsuario to set
     */
    public void setIdUsuario(int idUsuario) {
        this.idUsuario = idUsuario;
    }

    /**
     * @return the tipoUsuario
     */
    public int getTipoUsuario() {
        return tipoUsuario;
    }

    /**
     * @param tipoUsuario the tipoUsuario to set
     */
    public void setTipoUsuario(int tipoUsuario) {
        this.tipoUsuario = tipoUsuario;
    }

    /**
     * @return the password
     */
    public String getPassword() {
        return password;
    }

    /**
     * @param password the password to set
     */
    public void setPassword(String password) {
        this.password = password;
    }

    /**
     * @return the estadoU
     */
    public int getEstadoU() {
        return estadoU;
    }

    /**
     * @param estadoU the estadoU to set
     */
    public void setEstadoU(int estadoU) {
        this.estadoU = estadoU;
    }
}
